src/Http/samples/MinimalSample/Program.cs

```diff
@@ -3,6 +3,7 @@
 
 using Microsoft.AspNetCore.Http.HttpResults;
 using Microsoft.AspNetCore.Mvc;
+using System;
```

Comment:
```suggestion
```

Starting in .NET 6, `System` is already added by `<Project Sdk="Microsoft.NET.Sdk.Web">` as an [implicit global using directive](https://docs.microsoft.com/en-us/dotnet/core/project-sdk/overview#implicit-using-directives).